tests/helpers.py

```diff
@@ -0,0 +1,17 @@
+import sys
+import threading
+import time
+import traceback
+
+
+def debug_threads(wait_time: float = 1):
```

Comment:
neat helper that assisted me in figuring out why a test was hanging. Thanks cursor